superset-frontend/src/components/Table/index.tsx

```diff
@@ -134,6 +146,32 @@ export interface TableProps extends AntTableProps<TableProps> {
    * when the number of rows exceeds the visible space.
    */
   height?: number;
+  /**
+   * Sets the table to use react-window for scroll virtualization in cases where
+   * there are unknown amount of columns, or many, many rows
+   */
+  virtualize?: boolean;
+  /**
+   * Used to override page controls total record count when using server-side paging.
+   */
+  recordCount?: number;
+  /**
+   * Invoked when the tables sorting, paging, or filtering is changed.
+   */
+  onChange?: (
```

Comment:
Can you split this into different events? We may have different behaviors depending on which property changed.